hudi-gcp/src/main/java/org/apache/hudi/gcp/bigquery/HoodieBigQuerySyncClient.java

```diff
@@ -124,7 +128,7 @@ public void createOrUpdateTableUsingBqManifestFile(String tableName, String bqMa
       QueryJobConfiguration queryConfig = QueryJobConfiguration.newBuilder(query)
           .setUseLegacySql(false)
           .build();
-      JobId jobId = JobId.newBuilder().setProject(projectId).setRandomJob().build();
+      JobId jobId = JobId.newBuilder().setProject(billingProjectId).setRandomJob().build();
```

Comment:
Why is the billing project ID used here only?

Reply:
Got it, so it should only be used by`createOrUpdateTableUsingBqManifestFile`.  Could we rename the config to something like `--create-table-project-id` and `hoodie.gcp.bigquery.sync.create_table.project_id` to be specific?  Could you also update the config docs to reflect this?

Reply:
You can directly update
```
.withDocumentation("Name of the billing project id in BigQuery. By default it will use the BIGQUERY_SYNC_PROJECT_ID");
```
When we cut release docs, the config docs is going to be automatically updated.

Reply:
withDocumentation was already there. I will raise separate doc PR for updating syncing to catalogs.

Reply:
this one - https://hudi.apache.org/docs/next/gcp_bigquery